src/profile/notification-settings/utils.ts

```diff
@@ -0,0 +1,62 @@
+import { refreshSamUserAttributes, refreshTerraProfile } from 'src/auth/auth';
+import { Ajax } from 'src/libs/ajax';
+import Events, { EventWorkspaceAttributes, extractWorkspaceDetails } from 'src/libs/events';
+
+export const workspaceSubmissionNotificationKeys = (namespace: string, name: string) => [
+  `notifications/SuccessfulSubmissionNotification/${namespace}/${name}`,
+  `notifications/FailedSubmissionNotification/${namespace}/${name}`,
+  `notifications/AbortedSubmissionNotification/${namespace}/${name}`,
+];
+
+export const workspaceChangedNotificationKey = (namespace: string, name: string) => [
+  `notifications/WorkspaceChangedNotification/${namespace}/${name}`,
+];
+
+export const workspaceSubmissionNotificationInfo =
+  'Receive email notifications when a submission in this workspace has succeeded, failed, or been aborted.';
+export const workspaceChangedNotificationInfo =
+  'Receive email notifications when an owner of a workspace sends a workspace changed notification.';
+
+export type NotificationType =
```

Comment:
For MixPanel eventing, include a type so we can more easily see which options users are selecting. Previously we were eventing the "notificationKeys", but since these include the workspace namespace/name for workspace-level events, they don't lend themselves to easy filtering in MixPanel (plus there are 3 different keys for workspace submission).